Services/FtpEntries/FtpEntries.API/Controllers/FtpSyncController.cs

```
@@ -0,0 +1,80 @@
+        private readonly IMessageBrokerProducer<IPathInfo> producer;
+        private readonly IFactory<IPathInfo> pathInfoFactory;
+        private readonly ILogger<FtpSyncController> logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FtpSyncController"/> class.
+        /// </summary>
+        /// <param name="ftpPathRepository">Ftp path repository.</param>
+        /// <param name="producer">Message broker producer.</param>
+        /// <param name="pathInfoFactory">Path info factory.</param>
+        /// <param name="logger">Logger.</param>
+        public FtpSyncController(
+            IFtpPathRepository ftpPathRepository,
+            IMessageBrokerProducer<IPathInfo> producer,
+            IFactory<IPathInfo> pathInfoFactory,
+            ILogger<FtpSyncController> logger)
+        {
+            this.ftpPathRepository = ftpPathRepository ?? throw new ArgumentNullException(nameof(ftpPathRepository));
+            this.producer = producer ?? throw new ArgumentNullException(nameof(producer));
+            this.pathInfoFactory = pathInfoFactory ?? throw new ArgumentNullException(nameof(pathInfoFactory));
+            this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        }
+
+        /// <summary>
+        /// Creates a tender plan sync process.
+        /// </summary>
+        /// <returns>Void.</returns>
+        [HttpPost]
+        public async Task Create()
+        {
+            try
+            {
+                this.logger.LogInformation("Ftp syncronization starts...");
+                var paths = this.ftpPathRepository.Get().Select(c => this.MapToPathInfo(c, Guid.NewGuid())).AsQueryable(); // use process id
+
+                await this.producer
+                .SendAsync(paths.ToArray()).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                this.logger.LogCritical(e, e.Message);
```

Don't swallow Exception, rethrow it